controller/users.go

```diff
@@ -175,6 +194,11 @@ func (c *UsersController) Update(ctx *app.UpdateUsersContext) error {
 				return ctx.Conflict(jerrors)
 			}
 			identity.Username = *updatedUserName
+			keycloakUserProfile, err = c.copyExistingKeycloakUserProfileInfo(keycloakUserProfile, tokenString, accountAPIEndpoint)
```

Comment:
Thanks @hectorj2f 

That's what I had initially, the trouble with that was - keycloak was getting called even when we didnt need to ( `copyExistingKeycloakUserProfileInfo` calls keycloak ). `copyExistingKeycloakUserProfileInfo` needs to be called only for a few user profile attributes.

Example, when user contextInformation is being persisted, there is no need to have any keycloak interaction.